### ISSUE TITLE
bug(web): sentry-cli call in release build does not work (but is not failing build)

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyman#6516]
**Describe the bug**

Sample log from Release-14.0 for Keyman - Web:

https://build.palaso.org/buildConfiguration/Keymanweb_Build/315125?buildTab=log&focusLine=281&linesState=127

```
[04:04:32 ](https://build.palaso.org/buildConfiguration/Keymanweb_Build/315125?buildTab=log&focusLine=281&linesState=281)  /c/BuildAgent/work/fb3cd69e22dde528/keyman/web/release/embedded /c/BuildAgent/work/fb3cd69e22dde528/keyman/web/source
[04:04:32 ](https://build.palaso.org/buildConfiguration/Keymanweb_Build/315125?buildTab=log&focusLine=282&linesState=282)  Uploading to Sentry...
[04:04:33 ](https://build.palaso.org/buildConfiguration/Keymanweb_Build/315125?buildTab=log&focusLine=283&linesState=283)  
[04:04:33 ](https://build.palaso.org/buildConfiguration/Keymanweb_Build/315125?buildTab=log&focusLine=284&linesState=284)  > keyman@15.0.231 sentry-cli
[04:04:33 ](https://build.palaso.org/buildConfiguration/Keymanweb_Build/315125?buildTab=log&focusLine=285&linesState=285)  > sentry-cli "releases" "files" "release-15.0.231-beta" "upload-sourcemaps" "--strip-common-prefix" "release/embedded" "--rewrite" "--ext" "js" "--ext" "map" "--ext" "ts"
[04:04:33 ](https://build.palaso.org/buildConfiguration/Keymanweb_Build/315125?buildTab=log&focusLine=286&linesState=286)  
[04:04:33 ](https://build.palaso.org/buildConfiguration/Keymanweb_Build/315125?buildTab=log&focusLine=287&linesState=287)  error: failed to invoke sentry-cli
[04:04:33 ](https://build.palaso.org/buildConfiguration/Keymanweb_Build/315125?buildTab=log&focusLine=288&linesState=288)  Error: spawn C:\BuildAgent\work\fb3cd69e22dde528\keyman\web\node_modules\@sentry\cli\sentry-cli.exe ENOENT
[04:04:33 ](https://build.palaso.org/buildConfiguration/Keymanweb_Build/315125?buildTab=log&focusLine=289&linesState=289)      at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
[04:04:33 ](https://build.palaso.org/buildConfiguration/Keymanweb_Build/315125?buildTab=log&focusLine=290&linesState=290)      at onErrorNT (node:internal/child_process:477:16)
[04:04:33 ](https://build.palaso.org/buildConfiguration/Keymanweb_Build/315125?buildTab=log&focusLine=291&linesState=291)      at processTicksAndRejections (node:internal/process/task_queues:83:21)
[04:04:33 ](https://build.palaso.org/buildConfiguration/Keymanweb_Build/315125?buildTab=log&focusLine=292&linesState=292)  Upload successful.
```

---

**KeymanWeb:**
 - KeymanWeb Version: 15.0.231-beta

**Comment by jahorton** [comment]
15.0.180 - success
15.0.181 - success
15.0.182 - failure
15.0.185 - failure
15.0.190 - failure

14.0.286 (stable) - success - at a later date than some of those failures
14.0.290 (stable) - also success (end of March)

I can't find any successful `sentry-cli` calls after 15.0.181; therefore, there's a good chance something happened with 15.0.182 to break it.

**Comment by jahorton** [comment]
I can reproduce the behavior offline as well; `stable-14.0`'s version doesn't crash, but `beta`'s does - even with no parameters, which'd result in the display of help documentation.

**Comment by jahorton** [comment]
Significantly of note:  since I wasn't seeing any good leads, I looked at issues on https://github.com/getsentry/sentry-cli/ and noted that there were a few with similar error outputs, all of which were closed... so I figured "why not?" and decided to try updating the dependency to see if that'd fix things.

NOTE:  the CI servers may need to have their installed Node version updated.  Requires Node 12, and I forget which minor version of that was a minimum requirement.

That said, updating to `@sentry/cli` version `2.0.2` eliminated the locally-reproduced error.  Of course, there is a small chance it was actually updating my local Node version.  Either way, it sounds like it may be easily resolved via updating the toolchain a smidge.
